spec/config_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe 'configuration' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.